org.freedesktop.Piper.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "org.freedesktop.Piper",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "piper",
+      "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
This requires access to DConf, see https://github.com/flatpak/flatpak-docs/blob/master/docs/sandbox-permissions.rst#dconf-access. I learned this when submitting my own application to Flathub: https://github.com/flathub/flathub/pull/601#discussion_r214522390